vortex-array/src/arrays/list/compute/is_constant.rs

```diff
@@ -0,0 +1,135 @@
+use vortex_error::VortexResult;
+use vortex_scalar::NumericOperator;
+
+use crate::arrays::{ListArray, ListVTable};
+use crate::compute::{IsConstantKernel, IsConstantKernelAdapter, IsConstantOpts, numeric};
+use crate::register_kernel;
+
+impl IsConstantKernel for ListVTable {
+    fn is_constant(&self, array: &ListArray, opts: &IsConstantOpts) -> VortexResult<Option<bool>> {
+        // At this point, we're guaranteed:
+        // - Array has at least 2 elements
+        // - All elements are valid (no nulls)
+
+        if opts.is_negligible_cost() {
+            return Ok(None);
+        }
+
+        let start_offsets = array.offsets.slice(0, array.len())?;
```

Comment:
I wonder if it is worth checking the first few list lens or values of index zero as a fast exit.